FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m

```diff
@@ -327,20 +327,22 @@ - (void)URLSession:(NSURLSession *)session
 }
 
 - (void)receiveWebSocketData {
+    __weak FWebSocketConnection *weakSelf = self;
     [self.webSocketTask receiveMessageWithCompletionHandler:^(
                             NSURLSessionWebSocketMessage *_Nullable message,
                             NSError *_Nullable error) {
+      FWebSocketConnection *strongSelf = weakSelf;
```

Comment:
nit: I think the common pattern to exit early if `strongSelf == nil`, but it'll be fine if we don't in this particular case.